mock/datasets/no2.data.mdx

```diff
@@ -55,6 +55,20 @@ layers:
       id: polarNorth
     bounds: [-10, 36, -5, 42]
     description: Levels in 10¹⁵ molecules cm⁻². Darker colors indicate higher nitrogen dioxide (NO₂) levels associated and more activity. Lighter colors indicate lower levels of NO₂ and less activity.
+    info:
+      description: |
+        ::markdown 
+        ### Title
+
+        - list item 1
+        - lis item 2
+
+        [A tag test](https://developmentseed.org)
+      table:
+        source: NASA
+        spatialExtent: Global
+        latency: Monthly
+        unit: unit
```

Comment:
We will want to have nicer names for the keys - capitalized, with spaces and possibly unicode chars, etc.

```suggestion
        Source: NASA
        Spatial extent: Global
        Latency: Monthly
        Units: 10¹⁵ molecules cm⁻²
```

If that does not work nicely with the current object structure in YAML, perhaps we can use something like this?

```yaml
        - title: Source
          value: NASA
        - title: Spatial extent
          value: Global
```

or

```yaml
        source:
          title: Source
          value: NASA
        spatialExtent: 
          title: Spatial extent
          value: Global
```

Reply:
This change is fine but to clarify my understanding: 
Maybe my sandbox example was confusing (that the example renders the key name) from the current design, I only see this table data will be used programmatically (key will never be rendered directly, and will be used only to fetch the data from the layer.) Considering we offer description as markdown (which this information will be included as markdown list), do we need to make key name human-friendly?